### PR TITLE
Add Travis-CI supprt for continuous integration

### PR DESCRIPTION
**Pull request by minggangw**
To support Travis-CI, add .travis.yml to trigger and configure it.

### CODE REVIEW

**Comment by minggangw** [comment]
@halton @jondong please review the PR, thanks!

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+env:
+  matrix:
+  - XWALK_CHANNEL=canary XWALK_VERSION=18.46.466.0
+
+before_install:
+- cd ..
+
+install:
+- git config --global user.email "minggangw@gmail.com"
```
Comment (halton) [inline .travis.yml]: Could you use intel email instead? ditto as below

**Comment by halton** [comment]
LGTM with the nit comments

File: .travis.yml (hunk 1 above)
Comment (minggangw) [inline .travis.yml]: Will use intel email.  The other one is used as the account of Github, so I have to use the gmail account.

Comment (Bysmyyr) [inline .travis.yml]: You can add Intel email tu github in same time, then both works same way.

Comment (minggangw) [inline .travis.yml]: Got it, thanks!@Bysmyyr

**Pull request merged**